### ISSUE TITLE
What is the loss funciton used by the model?

### ISSUE THREAD

**Issue by CoCoNuTeK** [amazon-science/chronos-forecasting#121]
Hello there, 
I would like to ask about the loss function given i wanted to create my own loss function lets say moving average MASE as loss function to the model, everything is there except the outputs of the model contains: loss, logits and other stuff but there are no predicted values directly, 
so is there a way to use the tokenizer used for creating the input_ids, labels and attention_mask to somehow turn the logits into the predicted values, so the reverse operation?
